transaction/src/view/transaction_perspective.rs

```diff
@@ -6,6 +6,7 @@ use std::collections::BTreeMap;
 
 /// This represents the data to understand an individual transaction without
 /// disclosing viewing keys.
+#[derive(Debug, Clone, Default)]
```

Comment:
I was surprised that the `Default` impl was missing, I thought we'd added it a while ago on realization that the "default" TxP is the "exterior view" of the implementation.